scripts/ledger.lic

```diff
@@ -98,6 +106,25 @@ module Ledger
     end
   end
 
+  module Utility

```

Comment:
Not sure if these functions were really worth pulling into own module, was more when I added but got removed after I made further changes

Reply:
Should be fine to leave as a sub-module. It makes sense to me and more organization the better in my book.